conan/tools/env/environment.py

```diff
@@ -184,31 +196,66 @@ def define_path(self, name, value):
     def unset(self, name):
         """
         clears the variable, equivalent to a unset or set XXX=
+
+        :param name: Name of the variable to unset
         """
         self._values[name] = _EnvValue(name, None)
 
     def append(self, name, value, separator=None):
+        """
+        Append the `value` to an environment variable `name`
+
+        :param name: Name of the variable to append a new value
+        :param value: New value
+        :param separator: The character to separate the appended value with the previous value
+        """
         self._values.setdefault(name, _EnvValue(name)).append(value, separator)
 
     def append_path(self, name, value):
+        """
+        Similar to "append" method but indicating that the variable is a filesystem path.
```

Comment:
```suggestion
        Similar to "append" method but indicating that the variable is a filesystem path. It will automatically handle the path separators depending on the operating system.
```